### PR TITLE
Added Omen Testing DAO with Schemes

### PR DESCRIPTION
**Pull request by nicoelzer**
Added Omen Testing DAO with Schemes for Rinkeby network

### CODE REVIEW

File: daos/rinkeby/omentesting.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "Reputation": "0xcae8cb5d5d9ceefd2a5f3e56fb33e6104e552bd8",
+  "Controller": "0xc2103ff0ae40dcbf4a61009998d96e2dd1b5cb75",
+  "Schemes": [
+    {
+      "name": "GenericScheme",
+      "alias": "GenericSchemeOmenDisputeResolution",
+      "address": "0x67883470ae73347a6742376d3db8bf13bc9358a8"
+    },
+    {
+      "name": "ExchangeTokenRegistry",
```
Comment (orenyodfat) [inline daos/rinkeby/omentesting.json:14]: this also a genericscheme ?

File: daos/rinkeby/omentesting.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      "alias": "ExchangeTokenRegistry",
+      "address": "0xF86ab09C6a17c48B535D29fB69F95875c4856658"
+    },
+    {
+      "name": "MixTokenRegistry",
```
Comment (orenyodfat) [inline daos/rinkeby/omentesting.json:19]: this also a genericscheme ?

File: daos/rinkeby/omentesting.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      "alias": "MixTokenRegistry",
+      "address": "0x92d468e25bbBe47d0d266A58d0eb1E51f2276893"
+    },
+    {
+      "name": "OmenTokenRegistry",
```
Comment (orenyodfat) [inline daos/rinkeby/omentesting.json:24]: this also a genericscheme ?

File: daos/rinkeby/omentesting.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      "alias": "OmenTokenRegistry",
+      "address": "0x1BB96C24CbA4522810d9129db2e5d24546241F24"
+    }
+  ],
+  "arcVersion": "0.0.1-rc.30"
```
Comment (orenyodfat) [inline daos/rinkeby/omentesting.json:29]: why arc v 30?

File: daos/rinkeby/omentesting.json (hunk 1 above)
Comment (nicoelzer) [reply daos/rinkeby/omentesting.json:14]: @orenyodfat yes all the added schemes are Generics Schemes

**Comment by orenyodfat** [comment]
@nicoelzer could you write a brief description of this dao ?
what and why ...

**Pull request merged**